### ISSUE TITLE
Add means to remove tab-navigation

### ISSUE THREAD

**Issue by gisostallenberg** [sulu/sulu#2806]
| Q | A
| --- | ---
| Bug? | no
| New Feature? | yes
| Sulu Version | n/a
| Browser Version | n/a

#### Actual Behavior

One can add navigation following the [Using tab navigation](http://docs.sulu.io/en/latest/cookbook/using-tab-navigation.html) cookbook entry, but the entire service has to be overridden to remove an item. 

#### Expected Behavior

It would be nice when it was easier to remove a tab.

#### Possible Solutions

Maybe this can be accomplished just by allowing to change permissions for the tab.

**Comment by wachterjohannes** [comment]
in an ideal world every tab should be bound to a security-context. if you take a role this permissions you can hide it from a group of users. but in my opinion we should think about that each of the provider can return a list of tabs which shouldnt be displayed. for that we could use the action for example. /cc @danrot what do you think?

**Comment by danrot** [comment]
Yeah, the point is that it is currently not possible to exclude a tab completely, and not only hide it with the permissions. Returning another list would be a solution, but I am currently not sure if it is the best one.

**Comment by alexander-schranz** [comment]
This could now easy be implemented by adding a `remove` function to the [NavigationItemCollection](https://github.com/sulu/sulu/blob/2.0.0/src/Sulu/Bundle/AdminBundle/Admin/Navigation/NavigationItemCollection.php). Same for the RouteCollection.

**Comment by danrot** [comment]
Right, that would be quite easy to implement now. Gonna label this as an easy fix.

**Comment by Prokyonn** [comment]
Implemented in https://github.com/sulu/sulu/pull/7799
